java/memory/memory-netty/src/main/java/io/netty/buffer/UnsafeDirectLittleEndian.java

```diff
@@ -60,9 +59,6 @@
 
   private UnsafeDirectLittleEndian(AbstractByteBuf buf, boolean fake) {
     super(buf);
-    if (!NATIVE_ORDER || buf.order() != ByteOrder.BIG_ENDIAN) {
```

Comment:
Should keep the second check that `buf.order() != ByteOrder.BIG_ENDIAN` to make sure the wrapped buffer is BE?

Reply:
Sure, I will update this to check it using `buf.order !=ByteOrder.nativeOrder()`.

Reply:
I think that this is not necessary since the overriding accessors read/write data using `PlatfoormDependant.***`` directory. The buffer order does not matter.

Reply:
Yeah, from what I can tell the buffer order is only taken into account when calling `PlatfoormDependant.getIntSafe()` etc., which is not used by this.